includes/bash_profile

```diff
@@ -1,25 +1,22 @@
 # ~/.bash_profile: executed by bash(1) for interactive shells.
 
+source /opt/openhabian/functions/openhab.bash
+
 if [ -f "/opt/openHABian-install-failed" ]; then
-  echo ""
-  echo -e -n "\e[31;01mAttention! \e[39;49;00m"
+  echo -e -n "\\n\e[31;01mAttention! \e[39;49;00m"
   echo "The openHABian setup process seems to have failed on your system."
   echo "Sorry, this shouldn't happen! Please restart the installation process. Chances"
-  echo "are high setup will succeed on the second try."
-  echo ""
+  echo -e "are high setup will succeed on the second try.\\n"
   echo "In order to find the cause of the problem, have a look at the installation log:"
-  echo -e "   \e[90;01msudo cat /boot/first-boot.log\e[39;49;00m"
-  echo ""
+  echo -e "   \e[90;01msudo cat /boot/first-boot.log\e[39;49;00m\\n"
   echo "If the error persists, please read the debug guide at:"
-  echo -e "\e[94;04mhttps://github.com/openhab/openhabian/blob/master/docs/openhabian-DEBUG.md\e[39;49;00m"
-  echo ""
+  echo -e "\e[94;04mhttps://github.com/openhab/openhabian/blob/master/docs/openhabian-DEBUG.md\e[39;49;00m\\n"
   return 0
 elif [ -f "/opt/openHABian-install-inprogress" ]; then
-  echo ""
-  echo -e -n "\e[36;01mAttention! \e[39;49;00m"
+  echo -e -n "\\n\e[36;01mAttention! \e[39;49;00m"
   echo "The openHABian setup process is not finished yet."
```

Comment:
_has_ not ? You're the native speaker.

Reply:
I didn't look at the text, I just simplified the output code. I'll fix.

> LGTM
> however things I'm unaware of are
> 
> * today there already is something done to replace(?) .bash_profile on OH install/up-/downgrade. Would you need to remove anything of that ?

As of right now, I am inclined to leave it so that hopefully it gets replaced again with the new fixed version and then remove it in a couple of weeks.

> * how and when is .bash_profile created ?

On original installation, and for now on branch upgrade/switch.

>   can or should we do anything about it w.r.t. existing installations ? put a check into openhabian-config startup maybe ?

see above.

>   Not sure but be aware this is sensible and easily confuses people whenever there's a change.
>   Ideally we would not even _overwrite_ .bash_profile but allow the user to edit it.

After this whole mess is overwith, that should no longer be an issue as it will no longer be ovewritten because once I remove the code to replace it every time it will only be modified on original installation.


Any reason you are creating a duplicate PR?

Reply:
ok. The PR is your code just not against master as I wanted to test it upfront (but got interrupted)

Reply:
I can just push my code to a new branch if you would like. Do you ant it based off of OH3 or OH2

Reply:
Oh dear I though I was reviewing your PR only noticed now that it was actually my _copy_ of your PR I was reviewing.
So please take this as the input, fix the text according to my 2 comments above here then wen can merge it.